frontend/src/views/web/change-password.html

```diff
@@ -6,17 +6,23 @@
                 <div class="pass-title">Change Password</div>
                 <form name="changepwdForm" ng-submit="changepwd.changePassword(changepwdForm.$valid)" novalidate>
                     <div class="input-field align-left">
-                        <input id="old_password" name="old_password" type="password" class="text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" ng-minlength="8" required focus-if>
-                        <span class="form-icon form-icon-dark"><i class="fa fa-key"></i></span>
+                        <input id="old_password" name="old_password" type="{{canShowOldPassword ? 'text' : 'password'}}" class="text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" ng-minlength="8" required focus-if>
```

Comment:
Please revert all changes in this file. Not related to oauth integration. Feel free to open a new PR.